pymedphys/labs/winstonlutz/createaxis.py

```diff
@@ -25,14 +25,8 @@
 
 import numpy as np
 
-import matplotlib.transforms
-
-
-def transform_axis(x, y, translation, rotation):
-    transform = matplotlib.transforms.Affine2D()
-    transform.rotate_deg(-rotation)
-    transform.translate(*translation)
 
+def transform_axis(x, y, transform):
     transformed_x = transform @ np.vstack([x, np.zeros(len(x)), np.ones(len(x))])
```

Comment:
I'd never seen this @ operator for matrix multiplication before.. Cool!

Reply:
Yeah it is neat :)